style.css/fourthsectionStyle.css

```diff
@@ -0,0 +1,31 @@
+.fourth-section {
+  text-align: center;
```

Comment:
In this section, in the design, we have two item but here I can see just one.

Reply:
Fixed it.